src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIOFalcon.kt

```diff
@@ -220,7 +363,7 @@ class SwerveModuleIOFalcon(
   }
 
   override fun zeroSteering() {
```

Comment:
clear the queue when you zero, because if queue is not empty, that stuff will all still go into pose est